navapbc/wic-participant-recertification-portal-demo
# Create Participant Demo Site

## Ticket

https://wicmtdp.atlassian.net/browse/PRP-346

## Changes
* removed other environments from the deploy workflows
* update demo mode banner
* remove client side references to Matomo
* enable demo mode in the containers

## Context for reviewers
We need to create a version of the participant recertification portal for demo purposes.


## Testing
local testing:
1. Navigate to the `participant` directory and run the participant portal locally (`docker compose up -d`) 
2. Make sure the Demo mode banner is up on all pages (example below)
<img width="1318" alt="Screen Shot 2023-07-10 at 10 23 06 AM" src="https://github.com/navapbc/wic-participant-recertification-portal-demo/assets/37313082/b72a0ce0-eeb7-4626-96cf-a3b039d62622">

Live testing
